air_quality_monitoring/raqmn/raqmn_api/views.py

```diff
@@ -76,4 +68,35 @@ def get_average_data_12_hrs(request, uuid):
 		else:
 			return Response("Data unavailable")
 
+@csrf_exempt
+@api_view(['POST'])
+def post_sensor_data(request, key):
+	"""
+	Posts the data of the sensor to giles server
+	"""
+	if request.method == 'POST':
+		giles_url = 'http://localhost:8079/add/MYAPIKEY'
+		response = requests.post(giles_url, data=json.dumps(request.data))
+		# print request.data
+		return Response(response.content)
+
+def get_all_sensors():
+	"""
+	Returns the data of all sensors
+	"""
+	query_url = 'http://localhost:8079/api/query'
+	query = "select *"
+	r = requests.post(query_url, query)
+	return r.content
+
+def get_data_by_uuid_helper(uuid):
+	"""
+	Returns the data of sensor specified by uuid of last 30 days
```

Comment:
Right, so please update the documentation to reflect this to reduce confusion in the future.